src/status_im/ui/screens/intro/views.cljs

```diff
@@ -62,18 +64,28 @@
                                 :justify-content :flex-end
                                 :align-items :center
                                 :padding-horizontal 32}}
-            (let [margin 32
-                  size (min @width @height) #_(- (min @width @height) #_(* 2 margin))]
+            (let [size (min @width @height)]
               [react/view {:style {:flex 1}
                            :on-layout (fn [^js e]
-                                        (reset! height (-> e .-nativeEvent .-layout .-height)))}
+                                        (let [new-height (-> e .-nativeEvent .-layout .-height)]
+                                          (swap! height #(if (pos? %) (min % new-height) new-height))))}
                [react/image {:source (:image s)
                              :resize-mode :contain
                              :style {:width size
                                      :height size}}]])
             [react/i18n-text {:style styles/wizard-title :key (:title s)}]
-            [react/i18n-text {:style styles/wizard-text
-                              :key   (:text s)}]]))]
+            [react/text {:style (styles/wizard-text @text-height)
+                         :on-layout
+                         (fn [^js e]
+                           (let [new-height (-> e .-nativeEvent .-layout .-height)]
+                             (when (and (not= new-height @text-temp-height)
+                                        (not (zero? new-height))
+                                        (< new-height 200))
+                               (swap! text-temp-height #(if (pos? %) (max % new-height) new-height))
+                               (when @text-temp-timer (js/clearTimeout @text-temp-timer))
+                               (reset! text-temp-timer
+                                       (js/setTimeout #(reset! text-height @text-temp-height) 500)))))}
```

Comment:
Why the need to set a timeout here?

Reply:
because on-layout is called several times , we want to use the last one, so we use "debounce" with timer